pysb/export/sbml.py

```diff
@@ -90,84 +70,211 @@ class SbmlExporter(Exporter):
     Inherits from :py:class:`pysb.export.Exporter`, which implements
     basic functionality for all exporters.
     """
+    def __init__(self, *args, **kwargs):
+        super(SbmlExporter, self).__init__(*args, **kwargs)
+        if not libsbml:
+            raise ImportError('The SbmlExporter requires the libsbml python package')
 
-    def export(self):
-        """Export the SBML for the PySB model associated with the exporter.
+    def _sympy_to_sbmlast(self, sympy_expr, wrap_lambda=False):
+        """
+        Convert a sympy expression to the AST format used by libsbml
+
+        wrap_lambda indicates whether the expression should be converted into a mathml lambda function
+        """
+        mathml = print_mathml(sympy_expr)
```

Comment:
When converting to DOM building, this needs some thought. The sympy MathML printer returns a string which we'd have to run back through a parser to generate a DOM again. The XML strings should be short but it feels like a waste. The actual DOM object produced by the printer is behind the private method `Printer._print` so it feels a bit wrong to access that directly. And it looks like the `xml.dom.minidom` there isn't directly compatible with the DOM class in `lxml.etree`, so if we used `lxml` we'd have to round-trip anyway.